src/date-picker/calendar/date-picker-panel/component.ts

```diff
@@ -166,4 +170,68 @@ export class DatePickerPanelComponent extends CommonFormControl<Dayjs> {
   setToday() {
     this.confirmValue(dayjs(), true);
   }
+
+  clearValue() {
+    this.selectedTime = null;
+    this.clear.next();
+  }
+
+  private _disabledTimeFn(selectedDate: ConfigType) {
+    const getTimeFilter = (
+      date: Dayjs,
+      comparator: (a: number, b: number) => boolean,
+    ) => ({
+      hours: () => HOUR_ITEMS.filter(item => comparator(item, date.hour())),
+      minutes: (hour?: number) =>
+        hour === date.hour()
+          ? MINUTE_ITEMS.filter(item => comparator(item, date.minute()))
+          : [],
+      seconds: (hour?: number, minute?: number) =>
+        hour === date.hour() && minute === date.minute()
+          ? SECOND_ITEMS.filter(item => comparator(item, date.second()))
+          : [],
+    });
+
+    if (selectedDate) {
+      if (
+        this.minDate &&
+        (selectedDate as Dayjs)?.isSame(this.minDate, 'date')
```

Comment:
```suggestion
        (selectedDate as Dayjs).isSame(this.minDate, 'date')
```

这里不可能为空，另外为啥这里要 as 而不是改函数签名类型？

Reply:
DisabledTimeFn 的入参是 ConfigType，使用的地方确保了实参是 Dayjs ， 所以要 as 断言